### PR TITLE
Spectra V2.3.0

### PR DESCRIPTION
**Pull request by RajkiranBagal**
* New: Introducing the Slider block that allows you to make almost anything into a carousel, a good way to grab attention!
* Improvement: Added support for Astra's 4.0 theme menu position next to the Dashboard in admin.
* Fix: Container - Adding padding to paragraph cause uneven space and cross the boundary of container on responsive devices.
* Fix: Container - Background video goes out the boundary of the container if we have rounded corner.
* Fix: Image Gallery - When adding any of the HTML code or adding the link, the visibility of that caption gets changed to always visible
* Fix: Image Gallery - Broken links were shown in the gallery on frontend when added to caption.
* Fix: Icon List - Disable link for the list item, it turns on the toggle again.
* Fix: Testimonial - Carousel dots overlapping on below blocks in the mobile device.
* Fix: Post Grid - Undefined array key issue.
* Fix: How-To - Image is not visible when image selected from Image control.
* Fix: Image - Does not wrap with paragraph.
* Fix: Tabs - Container get hidden inside tabs body whenever use.
* Fix: Global - Buttons - Removed the WP default allowedFormats from the toolbar.
* Fix: Global - SVG Icons for blocks look big after switching to responsive devices inside the editor.
* Deprecated: Global - Removed deprecated toolbar and multiline for rich text.

### CODE REVIEW

File: includes/blocks/slider-child/attributes.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Attributes File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/slider-child/attributes.php:5]: it should be 2.3.0

File: includes/blocks/slider-child/block.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Block Information.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/slider-child/block.php:5]: Same here should be 2.3.0

File: includes/blocks/slider-child/frontend.css.php [hunk 3]
```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Slider child front end style
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/slider-child/frontend.css.php:5]: same here

File: includes/blocks/slider/attributes.php [hunk 4]
```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Attributes File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/slider/attributes.php:5]: same here

File: includes/blocks/slider/frontend.js.php [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Frontend JS File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/slider/frontend.js.php:5]: same here

**Review by patilvikasj — APPROVED** [review]

File: src/blocks/slider-child/save.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+/**
+ * BLOCK: Container - Save Block
```
Comment (AbhijitNage123) [inline src/blocks/slider-child/save.js:2]: change name to slider

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**